mkversion.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
+set -e
+cd "$(dirname "$0")"
+isdeb=true
+v="$( head -n1 debian/changelog  | sed -r -e 's/^[^ ]+ \((.*)\)( +[^ ]+)+; urgency=.*/\1/' )"
+if [ -z "$DH_GOPKG" ]; then
+    isdeb=false
+    # not building the package; add something to the version
+    if git rev-parse --is-inside-work-tree >&- 2>&-; then
```

Comment:
This will always be true for me (my $HOME has files inside a git tree).

The existence of $CWD/.git would be a better indicator.